packages/tasks/src/tasks/automatic-speech-recognition/data.ts

```diff
@@ -8,7 +8,7 @@ const taskData: TaskDataCustom = {
 		},
 		{
 			description: "An English dataset with 1,000 hours of data.",
-			id: "librispeech_asr",
+			id: "openslr/librispeech_asr",
```

Comment:
Maybe we can highlight our latest dataset here: https://huggingface.co/parler-tts/parler-tts-large-v1

It's fully open source + 45K hours of audio, significantly superior than Librispeech.

Reply:
Sorry meant this: https://huggingface.co/datasets/parler-tts/mls_eng